tests/unit-tests/test-class-usage-tracking.php

```diff
@@ -1,28 +1,31 @@
 <?php
 
 class Sensei_Usage_Tracking_Test extends WP_UnitTestCase {
-	public function setup() {
-		parent::setup();
-		$this->factory = new Sensei_Factory();
+	public function setUp() {
+		parent::setUp();
 		$this->usage_tracking = new Sensei_Usage_Tracking();
 	}
 
-	public function teardown() {
-		parent::teardown();
+	public function tearDown() {
```

Comment:
If you're not doing anything in the `tearDown` function, then you don't need to include it. `parent::teardDown` gets called implicitly.